__tests__/api.test.js

```diff
@@ -34,22 +34,21 @@ describe("GET /api/articles/:article_id && Testing error handling", () => {
     test("Returns an object with the article found, the article should contain all 6 properties", () => {
         return request(app).get("/api/articles/1").expect(200).then((res) => {
             const { article } = res.body;
-
             expect(article).toMatchObject({
                 author : expect.any(String),
                 title : expect.any(String),
                 article_id : expect.any(Number),
```

Comment:
This would probably need to be 1, to check you are getting back the correct object. At the moment, every object returned will pass this